src/Misc/Part.cpp

```diff
@@ -1246,7 +1246,8 @@ void Part::add2XML(XMLwrapper& xml)
     xml.addparbool("note_on", Pnoteon);
     xml.addparbool("poly_mode", Ppolymode);
     xml.addpar("legato_mode", Plegatomode);
-    xml.addpar("key_limit", Pkeylimit);
```

Comment:
atoi() parses an integer. Are there any problems about that?
Should you use "int" ??

Reply:
Ah, you've got a build where POLYPHONY is more than 127, right? In that case it should be easy to change `unsigned char keylimit_max` to `int keylimit_max`

Reply:
Sorry, this was already fixed in #235 , but this time at loading time, not at saving time. So this PR is deprecated.

Also, in #235 , I indeed used "int" here, so it should be fine.